edge/Makefile

```diff
@@ -58,4 +58,6 @@ armv8:
 .PHONY: small_build
 small_build:
 	go build -ldflags="-w -s -extldflags -static" cmd/edge_core.go
-	upx-ucl --best edge_core
+	# There are options in upx-ucl like --best, but it has a drawback of taking too long for big files, we are using --9 since it takes less time
```

Comment:
Need reminder to user that --best has best compression, how about like below? :)
`#` There are options in upx-ucl like --best, which has best compression,
`#` but also has a drawback of taking too long for big files, we are using `-9` since it takes less time

Reply:
Done @fisherxu, thanks :)